src/plugins/workspace/public/components/workspace_detail/workspace_detail_form_content.tsx

```diff
@@ -138,6 +146,38 @@ export const WorkspaceDetailFormContent = ({
           />
         </EuiCompressedFormRow>
       </EuiDescribedFormGroup>
+      <EuiDescribedFormGroup
+        title={<h3>{workspacePrivacyTitle}</h3>}
+        description={
+          <FormattedMessage
+            id="workspace.form.details.panels.privacy.description"
+            defaultMessage="Manage who can view or edit workspace and assign workspace administrators on the {collaboratorsLink} page."
+            values={{
+              collaboratorsLink: (
+                <EuiLink onClick={() => application.navigateToApp('workspace_collaborators')}>
```

Comment:
Can we use `WORKSPACE_COLLABORATORS_APP_ID` here? Shall we add some unit test to cover this line?